package.json

```diff
@@ -1,30 +1,33 @@
 {
-    "name": "@lingui/swc-plugin",
-    "version": "0.2.0",
-    "description": "A SWC Plugin for LinguiJS",
-    "author": {
-        "name": "Timofei Iatsenko",
-        "email": "ty@road.travel"
-    },
-    "license": "MIT",
-    "keywords": [
-        "swc-plugin",
-        "swc",
-        "nextjs",
-        "lingui",
-        "lingui-js",
-        "icu",
-        "message-format",
-        "i18n",
-        "internalization"
-    ],
-    "main": "target/wasm32-wasi/release/lingui_macro_plugin.wasm",
-    "exports": {
-        ".": "./target/wasm32-wasi/release/lingui_macro_plugin.wasm"
-    },
-    "scripts": {
-        "prepublishOnly": "cargo build-wasi --release"
-    },
-    "files": [],
-    "packageManager": "yarn@3.3.1"
+  "name": "@lingui/swc-plugin",
```

Comment:
Please also add the `repository` and `bugs` fields to let NPM display these links here https://www.npmjs.com/package/@lingui/swc-plugin